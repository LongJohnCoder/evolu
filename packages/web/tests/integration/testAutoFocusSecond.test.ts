import { pageDom, pageGoto } from './helpers';

beforeEach(async () => {
  await pageGoto('testAutoFocusSecond');
});

test('render', async () => {
  await expect(await pageDom()).toMatchSnapshot();
});
