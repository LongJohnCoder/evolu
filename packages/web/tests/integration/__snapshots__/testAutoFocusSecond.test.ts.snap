// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<div
  id="__next"
>
  <div
    class="jsx-442929638"
  >
    ·Basic Example·
  </div>
  <div
    autocorrect="off"
    contenteditable="true"
    data-gramm="true"
    role="textbox"
    spellcheck="false"
    style="background-color: rgb(238, 238, 238); margin-bottom: 24px; outline: none; padding: 8px; width: 300px;"
    tabindex="0"
  >
    <div
      class="root"
    >
      <div
        style="font-size: 24px;"
      >
        ·heading·
      </div>
      <div
        style="font-size: 16px;"
      >
        ·paragraph·
      </div>
    </div>
  </div>
  <pre
    class="jsx-2255726786"
  >
    <span
      class="jsx-1782790912"
      title="{'tag':'div','props':{'className':'root'},'children':[{'tag':'div','props':{'style':{'fontSize':'24px'}},'children':[{'text':'heading'}]},{'tag':'div','props':{'style':{'fontSize':'16px'}},'children':[{'text':'paragraph'}]}]}"
    >
      ·0·
      · ·
      ·{"hasFocus":false,"selection":null}·
    </span>
  </pre>
  <div
    class="jsx-442929638"
  >
    ·Schema Example·
  </div>
  <div
    autocorrect="off"
    contenteditable="true"
    data-gramm="true"
    role="textbox"
    spellcheck="false"
    style="background-color: rgb(238, 238, 238); margin-bottom: 24px; outline: none; padding: 8px; width: 300px;"
    tabindex="0"
  >
    <div
      class="css-0"
    >
      <div
        class="css-1s76os7-heading"
      >
        ·heading·
      </div>
      <div
        class="css-1mzu32r-paragraph"
      >
        ·paragraph·
      </div>
      <div
        class="css-1kkfp6h-list"
      >
        <div
          class="css-0"
        >
          ·listitem·
        </div>
      </div>
      <img
        alt="Square placeholder image 80px"
        height="80"
        src="https://via.placeholder.com/80"
        width="80"
      />
    </div>
  </div>
  <pre
    class="jsx-2255726786"
  >
    <span
      class="jsx-1782790912"
      title="{'type':'document','children':[{'type':'heading','children':[{'text':'heading'}]},{'type':'paragraph','children':[{'text':'paragraph'}]},{'type':'list','children':[{'type':'listitem','children':[{'text':'listitem'}]}]},{'type':'image','src':'https://via.placeholder.com/80','alt':'Square placeholder image 80px','width':80,'height':80,'children':[]}]}"
    >
      ·1·
      · ·
      ·{"hasFocus":true,"selection":{"anchor":[0,0,0],"focus":[0,0,0]}}·
    </span>
    <span
      class="jsx-1782790912"
      title="{'type':'document','children':[{'type':'heading','children':[{'text':'heading'}]},{'type':'paragraph','children':[{'text':'paragraph'}]},{'type':'list','children':[{'type':'listitem','children':[{'text':'listitem'}]}]},{'type':'image','src':'https://via.placeholder.com/80','alt':'Square placeholder image 80px','width':80,'height':80,'children':[]}]}"
    >
      ·0·
      · ·
      ·{"hasFocus":true,"selection":null}·
    </span>
  </pre>
  <div
    style="margin-bottom:24px"
  >
    <button
      class="focus"
      disabled=""
      tabindex="0"
      type="button"
    >
      ·focus·
    </button>
    <button
      class="blur"
      tabindex="-1"
      type="button"
    >
      ·blur·
    </button>
  </div>
</div>
`;
